src/systems/filecoin_mining/sector/sealing.id

```diff
@@ -22,20 +22,23 @@ type SealCfg struct {
 // needs to verify a Seal.
 type SealVerifyInfo struct {
     SectorID
-    OnChain     OnChainSealVerifyInfo
+    OnChain                OnChainSealVerifyInfo
     SealCfg
-    PieceInfos  [PieceInfo]
+    Randomness             SealRandomness
+    InteractiveRandomness  InteractiveSealRandomness
+    PieceInfos             [PieceInfo]
 }
 
 // OnChainSealVerifyInfo is the structure of information that must be sent with
 // a message to commit a sector. Most of this information is not needed in the
 // state tree but will be verified in sm.CommitSector. See SealCommitment for
 // data stored on the state tree for each sector.
 type OnChainSealVerifyInfo struct {
-    SealedCID     SealedSectorCID  // CommR
-    Epoch         block.ChainEpoch
-    Proof         SealProof
-    DealIDs       [deal.DealID]
+    SealedCID         SealedSectorCID  // CommR
+    Epoch             block.ChainEpoch
+    InteractiveEpoch  block.ChainEpoch
```

Comment:
- is this the `ChallengeEpoch`?  (`InteractiveEpoch` not super clear)
- can we name both epochs well to distinguish them?

Reply:
and maybe add one-line comment